packages/augur-ui/src/modules/modal/add-funds.tsx

```diff
@@ -213,27 +200,26 @@ export const AddFunds = ({
                   defaultSelection={SWAP_ID}
                 />
               </div>
-              {poolSelected && (
-                <Swap
-                  address={loginAccount.address}
-                  balances={loginAccount.balances}
-                  toToken={tokenToAdd}
-                  fromToken={initialSwapToken ? initialSwapToken : null}
-                  ETH_RATE={ETH_RATE}
-                  REP_RATE={REP_RATE}
-                  config={config}
-                  useSigner={useSigner}
-                />
-              )}
+
+              <Swap
+                address={loginAccount.address}
+                balances={loginAccount.balances}
+                toToken={tokenToAdd}
+                fromToken={initialSwapToken ? initialSwapToken : null}
+                ETH_RATE={ETH_RATE}
+                REP_RATE={REP_RATE}
+                config={config}
+                useSigner={useSigner}
+              />
             </>
           )}
 
           {selectedOption === ADD_FUNDS_CREDIT_CARD && (
             <CreditCard
```

Comment:
same comment as the swap above. AddFundsTorus and Fortmatic for example could just be defined inside of `creditCard` instead of here.